### PR TITLE
new package: argp-standalone

### PR DESCRIPTION
**Pull request by jprjr**
This is an implementation of argp, for libc implementations that lack it (musl)

### CODE REVIEW

**Comment by xtraeme** [comment]
There's no reason to name it `foo-devel`, just name it `argp-standalone`.

**Comment by jprjr** [comment]
No problem, just pushed a new, rebased commit with the correct name

**Comment by xtraeme** [comment]
- build_pie is unnecessary.
- wrksrc is set by default to `${pkgname}-${version}`, redundant.
- nostrip is unnecessary.

**Comment by jprjr** [comment]
Okey-doke - fix+rebased

File: srcpkgs/argp-standalone/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+revision=1
+build_style=gnu-configure
+short_desc="Implementation of ARGP"
+maintainer="John Regan <john@jrjrtech.com>"
+homepage="https://www.freshports.org/devel/argp-standalone/"
+license="Public Domain"
+distfiles="http://www.lysator.liu.se/~nisse/misc/argp-standalone-${version}.tar.gz"
+checksum=dec79694da1319acd2238ce95df57f3680fea2482096e483323fddf3d818d8be
+
+export CFLAGS="-fPIC"
```
Comment (xtraeme) [inline srcpkgs/argp-standalone/template]: There's no need to use `export`.

**Comment by jprjr** [comment]
Removed `export`

**Pull request merged**